draft-ietf-rats-corim.md

```diff
@@ -2489,6 +2500,17 @@ The profile must specify how to compare the CBOR tagged Reference Value against
 
 Note that a Verifier may compare Reference Values in any order, so the comparison should not be stateful.
 
+## Concise Software Identifier Extension {#sec-coswid}
+
+A CoRIM may contain a CoSWID tag as specified in {{-coswid}}.
+The CoRIM profile allows for the specifier to provide meaning to extension points in the "Specification Required" range of the extension points `$version-scheme` and `$$coswid-extension`.
```

Comment:
```suggestion
```
Is removal okay since I already describe essentially the same thing in the profile section above?